### PR TITLE
[FX-4908] Add 'Create Azure DevOps Work Item for Findings' page

### PR DESCRIPTION
**Pull request by kittaakos**
### Added

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
|Create Azure DevOps Work Item for Findings | [Link](https://deploy-preview-562--cobalt-docs.netlify.app/integrations/integrationbuilder/how-to-guides/azure-devops/) | New page |

### Updated

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
|How to Guides | [Link](https://deploy-preview-562--cobalt-docs.netlify.app/integrations/integrationbuilder/how-to-guides#application-specific-guides) | Add Azure DevOps to index page |

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[x] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Integrations/IntegrationBuilder/How to Guides/azure-devops.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/03-create-connection.png" alt="Create a new connection" %}}
+
+Search for the Azure DevOps connection and click on it.
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/04-select-azure-devops-connection.png" alt="Select Azure DevOps connection" %}}
+
+Fill in the connection details: name, recipe location, and select **Personal Access Token** as the OAuth 2.0 grant type. Set the API version to **7.0** and specify the Azure DevOps organization name, the email address of the user to impersonate, and the personal access token.
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/05-connection-details.png" alt="Connection details" %}}
```
Comment (mahmoudadly-cobalt) [inline content/en/Integrations/IntegrationBuilder/How to Guides/azure-devops.md:42]: Please show a dummy email or pixelate it.

File: content/en/Integrations/IntegrationBuilder/How to Guides/azure-devops.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+> **ⓘ** Refer to the Azure DevOps [documentation](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows#create-a-pat) to create a personal access token.
+
+> **ⓘ** Use `Read, write, & manage` scopes for this example. Learn more about OAuth [scopes](https://learn.microsoft.com/en-us/azure/devops/integrate/get-started/authentication/oauth?view=azure-devops#scopes) for Azure DevOps.
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/06-azure-devops-pat-scopes.png" alt="Azure DevOps PAT scopes" %}}
+
+You should now see the new connection in the project, and it is connected.
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/07-connection-connected.png" alt="Connection connected" %}}
```
Comment (mahmoudadly-cobalt) [inline content/en/Integrations/IntegrationBuilder/How to Guides/azure-devops.md:52]: Same note about the email.

File: content/en/Integrations/IntegrationBuilder/How to Guides/azure-devops.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+### Create a New Azure DevOps Recipe
+
+Go to the "Library" tab and search for the "create work item" recipe.
```
Comment (mahmoudadly-cobalt) [inline content/en/Integrations/IntegrationBuilder/How to Guides/azure-devops.md:56]: Searching for "work item" seems generic. Why not search for "Azure" or something Azure-specific?

File: content/en/Integrations/IntegrationBuilder/How to Guides/azure-devops.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/08-search-create-work-item.png" alt="Search create work item" %}}
+
+Choose the recipe based on whether you want to create Work Items for a pentest finding or a DAST Scanner finding. We'll use the pentest finding recipe as a base and customize it. Click on the recipe to see its details and click "Use this recipe".
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/09-use-recipe.png" alt="Use recipe" %}}
+
+Save the recipe in the folder you created earlier.
+
+{{% image src="/integrations/integration_builder/how_to_guides/azure-devops/10-recipe-location.png" alt="Recipe location" %}}
```
Comment (mahmoudadly-cobalt) [inline content/en/Integrations/IntegrationBuilder/How to Guides/azure-devops.md:66]: Another huge modal.

File: content/en/Integrations/IntegrationBuilder/How to Guides/azure-devops.md (hunk 3 above)
Comment (mahmoudadly-cobalt) [reply content/en/Integrations/IntegrationBuilder/How to Guides/azure-devops.md:56]: Actually "work item" is Azure-specific. 😅

File: content/en/Integrations/IntegrationBuilder/How to Guides/azure-devops.md (hunk 4 above)
Comment (kittaakos) [reply content/en/Integrations/IntegrationBuilder/How to Guides/azure-devops.md:66]: https://github.com/cobalthq/cobalt-product-public-docs/pull/562/commits/754c37f062f10de2500eeb8f5ef8ff14c2c5644f

File: content/en/Integrations/IntegrationBuilder/How to Guides/azure-devops.md (hunk 1 above)
Comment (kittaakos) [reply content/en/Integrations/IntegrationBuilder/How to Guides/azure-devops.md:42]: https://github.com/cobalthq/cobalt-product-public-docs/pull/562/commits/754c37f062f10de2500eeb8f5ef8ff14c2c5644f

**Comment by grayson-cobalt** [comment]
Nice!
Can we add a link to this guide from the table at the top of the "how to guides" index file please?
Thanks!

**Review by grayson-cobalt — APPROVED** [review]

**Review by grayson-cobalt — APPROVED** [review]
Thank you!

**Pull request merged**